pkg/apis/kops/validation/validation.go

```diff
@@ -984,6 +984,10 @@ func validateNetworkingCalico(v *kops.CalicoNetworkingSpec, e kops.EtcdClusterSp
 		allErrs = append(allErrs, validateCalicoAutoDetectionMethod(fldPath.Child("ipv6AutoDetectionMethod"), v.IPv6AutoDetectionMethod, ipv6.Version)...)
 	}
 
+	if v.WireguardEnabled && v.MajorVersion != "v3" {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("wireguardEnabled"), v.WireguardEnabled, fmt.Sprint("Unable to use wireguard except v3, please set v3 to majorVersion")))
+	}
+
```

Comment:
`v3` is the default now and `v2` will be removed in kops 1.20, so we can skip this.
```suggestion
```

Reply:
I removed validation and validation tests.